client/src/component/User.js

```diff
@@ -1,10 +1,29 @@
+import React, { useState } from "react";
+import "./User.css";
+
+
 export function User() {
 
+	const [name, setName] = useState("");
+
+	const handleInputChange = (event) => {
+		setName(event.target.value);
+	};
+
+	// const handleSearch = () => {
+	//   // Perform the search logic here using the "name" value
+	//   // You can make an API call or search the data locally
+
+	//   // Example: Logging the name value
+	//   console.log(`Searching for: ${name}`);
+	// };
+
 	return (
-			<div>
-				<h1>hello world</h1>
-			</div>
+<div className="user-container">
+	<input className="user-input" type="text" value ={name} onChange={handleInputChange} placeholder="Enter your full name" />
+	<br />
```

Comment:
can I know what is the porpos of this <br />

Reply:
what do you mean? do you mean <br/>?

Reply:
if you mean br it is Line Break element